src/modules/date/date.js

```diff
@@ -323,16 +323,16 @@
             return new Date(r[3], r[2] - 1, r[1]);
         } else if (str) {
             // parse relative date
-            var inc = str[2]*60*60*24*1000;
-            var timestamp = now.getTime();
+            var days = parseInt(str[2]);
```

Comment:
jo, už jsem ho tam přidal
2. 3. 2015 v 11:16, Jiri Dedic <notifications@github.com<mailto:notifications@github.com>>:


In src/modules/date/date.js<https://github.com/lmc-eu/ngx-library/pull/48#discussion_r25588493>:

> @@ -323,16 +323,16 @@
>              return new Date(r[3], r[2] - 1, r[1]);
>          } else if (str) {
>              // parse relative date
> -            var inc = str[2]*60*60*24*1000;
> -            var timestamp = now.getTime();
> +            var days = parseInt(str[2]);


pridej druhy parametr -> 10

—
Reply to this email directly or view it on GitHub<https://github.com/lmc-eu/ngx-library/pull/48/files#r25588493>.